docs/content/architectuur/zgw-in-gegevenslandschap.md

```diff
@@ -0,0 +1,188 @@
+Documenten bij een Verzoek worden bij het Verzoek vastgelegd. Later ook rechtstreeks gerelateerd aan de zaak?
+
+### 1.4.2. Waar slaan we procesinformatie op?
+
+Het is nog onduidelijk waar we in een gegevenslandschap “Procesinformatie” willen opslaan. In bovenstande figuur is dit – samen met procesdefinitiegegevens - gepositioneerd in de Zaakafhandelcomponent (net als nu in de diverse Taaksystemen).
+Dit is echter (mogelijk) in strijd met het uitgangspunt in Common ground dat functionaliteit en gegevens van elkaar gescheiden moeten zijn. Procesgegevens zijn immers ook gegevens.
+De procesdefinitie is in veel gevallen zelfs impliciet (hard gecodeeerde flow tussen schermen); dat valt dan niet eens los te knippen.
+
+Conclusie: hier is sprake van een glijdende schaal. Je wilt liefst alle domeingegevens en zo veel mogelijk procesgegevens uit de applicaties halen en via een gestandaardiseerde APi ontsluiten. We moeten echter wel oppassen dat we hier te ver in willen gaan en het paard daarmee achter de wagen spannen. In de praktijk zal dit voor veel bestaande taaksystemen een groeipad worden..
+
+Er is één maar. Het delen van procesinformatie is nodig om het handelen als overheid te verantwoorden: het moet dus gearchiveerd worden. Wie heeft op welke datum welke stap gezet? Welke procesflow (en keuzes) is er doorlopen? Welke informatie was wanneer beschikbaar? Hoe luidde de (deel)berekening die heeft geleid tot een bepaald resultaat? Dit is informatie die niet in één van de andere registraties past (tenzij als logfile in PDF in de DRC) en voor het grootste deel niet in de ZRC wordt vastgelegd (die kent meestal maar één behandelaar en vooral enkel het proces op hoofdlijnen: de Statussen).
+Dit pleit voor een procesverloopregistratie, waarin op een zo gestandaardiseerde manier als mogelijk informatie over het verloop van alle processen wordt bijgehouden. Probleem is echter dat het ene proces het andere niet is en het erg moeilijk is dit gestandaardiseerd te doen (een zaak probeert dat feitelijk ook, en deze kent niet voor niets enkel statussen). Dus wellicht krijg je dan een subsiedieprocesverloopregistratie, een vergunningprocesverloopregistratie, een burgerzakenprocesverloopregistratie, … Dat is dan weer niet aanlokkelijk.
+Wellicht dat een combinatie met de desbetreffende domeinregistratie dan nog het meest voor de hand ligt. Een domeinregistratie Vergunningen (en handhaving), waarin –volgens een gestandaardiseerd informatiemodel- naast de in het vergunningproces betrokken objecttypen, ook het verloop van de vergunningprocessen wordt geregistreerd.
+
```

Comment:
Wat is de relatie tussen deze redenering en de verschillende logging mechanismen uit het 'Gegevenslandschap vanuit gemeentelijk perspectief' die gebruikt worden om de rechtamtigheid van een handeling te bepalen?
Gaan we beiden doen: loggen en het procesverloop vastleggen? Zijn ze wel of niet completerend aan elkaar? En waar staan dan die log-gegevens?

Reply:
loggen in het gegevenslandschap doelt met name op het loggen van het (rechtmatig)gebruik van gegevens. Hier gaat het om 'loggen' van het gelopen proces IN een applicatie..dat is een aanvullend perspectief zou ik zeggen..